docs/e2e-testing.md

```diff
@@ -30,14 +30,7 @@ The test name should communicate the purpose and behaviour of the test. A clear
 - send 1 TST to Bob
 ```
 
-⚠️ Test names remain approachable, recommend to avoid using the meaningless prefix 'should'
-
-```javascript
-- should add Bob to the address book
-- should send 1 TST to Bob
-```
-
-❌ Test name should be completely avoided: The use of a `should` prefix and the word `and` can decrease the readability of the test,making it harder to understand what the test is doing as well as diagnose and fix issues.
+❌ Test name should be completely avoided: Long test names that may over explain the test intention and the word `and` can decrease the readability of the test,making it harder to understand what the test is doing as well as diagnose and fix issues.
```

Comment:
This new explanation for why the below test name should be avoided seems inaccurate. It's not a particularly long description, and it does not over explain the test intention.

There is also a missing space:
```suggestion
❌ Test name should be completely avoided: Long test names that may over explain the test intention and the word `and` can decrease the readability of the test, making it harder to understand what the test is doing as well as diagnose and fix issues.
```